x-pack/plugins/monitoring/server/routes/api/v1/cluster/clusters.ts

```diff
@@ -6,32 +6,30 @@
  */
 
 import { schema } from '@kbn/config-schema';
-import { LegacyRequest, LegacyServer } from '../../../../types';
 import { getClustersFromRequest } from '../../../../lib/cluster/get_clusters_from_request';
+import { getIndexPatterns } from '../../../../lib/cluster/get_index_patterns';
 import { verifyMonitoringAuth } from '../../../../lib/elasticsearch/verify_monitoring_auth';
 import { handleError } from '../../../../lib/errors';
-import { getIndexPatterns } from '../../../../lib/cluster/get_index_patterns';
+import { LegacyRequest, MonitoringCore } from '../../../../types';
 
-export function clustersRoute(server: LegacyServer) {
+export function clustersRoute(server: MonitoringCore) {
   /*
    * Monitoring Home
    * Route Init (for checking license and compatibility for multi-cluster monitoring
    */
 
   // TODO switch from the LegacyServer route() method to the "new platform" route methods
```

Comment:
is this TODO still relavant?

Reply:
Yes, this is still using a shim instead of calling the platform route registration directly. At some point we should probably make another pass to migrate to that. But in terms of value (type safety) I guess converting the client side to TS is more valuable as a next task.